shared/folders/files/index.js

```diff
@@ -45,12 +46,13 @@ class Files extends Component<void, Props, State> {
     const {folder, username} = this.props
     if (!folder) return null // Protect from state where the folder to be displayed was removed
     const openCurrentFolder = () => { this.props.openInKBFS(this.props.path) }
+    const ignoreCurrentFolder = () => { console.log('TODO (AW): implement ignore folder') } // TODO (AW): implement ignore file
     return (
       <Render
-        theme={folder.isPrivate ? 'private' : 'public'}
+        theme={folder.isPublic ? 'public' : 'private'}
```

Comment:
Hmm, after my rebasing adventure I'm not sure anymore. I'll look back through these and get the commits in order/squash them if it makes more sense.